Changelog

```diff
@@ -1,4 +1,83 @@
 Unreleased - sledgehammer999 <sledgehammer999@qbittorrent.org> - v4.5.0
+    - FEATURE: Add "Auto resize columns" functionality (Chocobo1)
+    - FEATURE: Allow to use Category paths in "Manual" mode (glassez)
+    - FEATURE: Allow to disable Automatic mode when "temp" path changed (glassez)
+    - FEATURE: Add tuning options related to performance warnings (Chocobo1)
+    - FEATURE: Add right click menu for status filters (An0n)
+    - FEATURE: Allow setting the number of maximum active checking torrents (An0n)
+    - FEATURE: Add option to toggle filters sidebar (AbeniMatteo)
+    - FEATURE: Allow to set "working set limit" on non-Windows OS (Chocobo1)
+    - FEATURE: Add "Export .torrent" action (Chocobo1)
+    - FEATURE: Add keyboard navigation keys (itlezy)
+    - FEATURE: Allow to use POSIX-compliant disk IO type (Coda)
+    - FEATURE: Add "Filter files" field in new torrent dialog (thalieht)
+    - FEATURE: Implement new icon/color theme (Nowshed H. Imran)
+    - FEATURE: Add file name filter/blacklist (mxtsdev, thalieht)
+    - FEATURE: Add support for custom SMTP ports (Emil M George)
+    - FEATURE: Split the OS cache settings into Disk IO read/write modes (summer)
+    - FEATURE: When duplicate torrent is added set metadata to existing one (glassez)
+    - FEATURE: Show startup progress dialog (glassez)
+    - FEATURE: Add keyboard shortcut to Download URL dialog (Chocobo1)
+    - FEATURE: Add ability to run external program on torrent added (glassez)
+    - FEATURE: Add infohash and download path columns (tristanleboss)
+    - FEATURE: Allow to set torrent stop condition (glassez, thalieht)
+    - FEATURE: Add a "Moving" status filter (tristanleboss)
+    - FEATURE: Introduce color palettes for both dark, light themes (Chocobo1)
+    - FEATURE: Add a "Use proxy for hostname lookup" option (Nathan Lewis)
+    - FEATURE: Introduce a 'change listen port' cmd option (BallsOfSpaghetti)
+    - BUGFIX: Handle .!qB extension more transparently (glassez)
+    - BUGFIX: Store hybrid torrents using "torrent ID" as basename (glassez)
+    - BUGFIX: Allow to open incomplete files from content view (glassez)
```

Comment:
It most likely fixes some intermediate bug so it shouldn't be mentioned here.

Reply:
commit 47daf1b358f7d2497991d7ebc83d956664b067a1

Reply:
> commit [47daf1b](https://github.com/qbittorrent/qBittorrent/commit/47daf1b358f7d2497991d7ebc83d956664b067a1)

So yes. It's just hotfix for regression since one of previous commits. Should not be mentioned here.